hmt_escrow/utils.py

```diff
@@ -0,0 +1,47 @@
+import time
+import logging
+
+
+logger = logging.getLogger("hmt_escrow.job")
+
+
+def with_retry(fn, retries=3, delay=5, backoff=2):
+    """ Retry a function
+
+    Mainly for BC transactions. Uses expnential backoff.
```

Comment:
what is BC transaction here?

Reply:
blockchain. will work on a better description for this method.